src/scripts/ethDeploy.sh

```diff
@@ -31,15 +31,15 @@ taraAddress=$(echo "$res" | grep "Deployed to:" | awk '{print $3}')
 
 echo "Deployed to: $taraAddress"
 
-echo "ETH_TARA_ADDRESS=$taraAddress" >> .env
+echo "TARA_ADDRESS_ON_ETH=$taraAddress" >> .env
 
 echo "Running deployment script for TaraClient & EthBridge"
 
-ethBridge=$(forge script src/scripts/Eth.deploy.s.sol:EthDeployer --via-ir --rpc-url $RPC_HOLESKY --broadcast)
+ethBridge=$(forge script src/scripts/Eth.deploy.s.sol:EthDeployer --via-ir --rpc-url $RPC_HOLESKY --broadcast --legacy)
```

Comment:
```suggestion
ethBridge=$(forge script src/scripts/Eth.deploy.s.sol:EthDeployer --via-ir --rpc-url $RPC_HOLESKY --broadcast )
```
I don't think we need legacy here, right?